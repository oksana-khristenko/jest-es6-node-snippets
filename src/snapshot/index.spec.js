import example from './';

describe('snapshot', () => {
  test('should return correct object', () => {
    expect(example(1,2,3)).toMatchSnapshot();
  });
});