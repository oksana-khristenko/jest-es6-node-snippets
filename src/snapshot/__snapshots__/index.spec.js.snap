// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot should return correct object 1`] = `
Object {
  "aa": 1,
  "bb": 2,
  "cc": 3,
}
`;
